control-plane/tests/test_service_persistence_action_reconciliation.py

```
@@ -16,6 +16,516 @@
 
 
 class ActionReconciliationPersistenceTests(ServicePersistenceTestBase):
+    def test_service_delegates_approved_create_tracking_ticket_through_shuffle(
+        self,
+    ) -> None:
+        store, _ = make_store()
```

Ruff (0.15.10)

[error] Undefined name `make_store`

(F821)